src/Nox.Generator/Application/Dto/EntityLocalizedDtoGenerator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.CodeAnalysis;
+using Nox.Generator.Common;
+using Nox.Solution;
+using System;
+
+namespace Nox.Generator.Application.Dto;
+
+internal class EntityLocalizedDtoGenerator : INoxCodeGenerator
+{
+    public NoxGeneratorKind GeneratorKind => NoxGeneratorKind.Domain;
```

Comment:
Dto are application layer
```suggestion
    public NoxGeneratorKind GeneratorKind => NoxGeneratorKind.Application;
```

Reply:
Agreed. Other generators on application level are also set as Domain kind. Should we change them as well?

Reply:
Looks like the files are not generated when we specify the NoxGeneratorKind.Application. Should this be investigated further?

Reply:
the test project needs to be set ptoperly the nox.generator.yaml

I would not do this in this ticket, because this needs to be reviewed for all layers

Reply:
just change this one to be correct

Reply:
The above issue applies for this one as well. The tests are failing because files are not generated.
![image](https://github.com/NoxOrg/Nox.Generator/assets/30667828/ffaf270c-1aaf-4042-b5f3-5137a2399c67)

Reply:
ok, I will take car of that later